oss/boost/MAINTAINER_README.md

```diff
@@ -0,0 +1,15 @@
+### Notes for Future Maintainers
+
+This was originally imported by @Austin-Lamb in December 2020.
+
+The provenance information (where it came from and which commit) is stored in the file `cgmanifest.json` in the same directory as this readme.
+Please update the provenance information in that file when ingesting an updated version of the dependent library.
+That provenance file is automatically read and inventoried by Microsoft systems to ensure compliance with appropiate governance standards.
+
+## What should be done to update this in the future?
+
+1. Download the version of boost you want from boost.org.
+2. Take the parts you want, but leave most of it behind since it's HUGE and will bloat the repo to take it all.  At the time of this writing, we only use small_vector.hpp and its dependencies as a header-only library.

```

Comment:
> we only use small_vector.hpp and its dependencies

holy shit it's 449 files for one header?! Man boost is giving npm a run for it's money

Reply:
Yeah, boost is...big.  The full boost zip, when unzipped, is over 600MB - so I tried to strip it down to a minimal set.  It could probably be more minimal than this but it felt hard to go smaller than whole folders of headers.

Reply:
holy _what_

Reply:
Sounds like boost needs C++20 modules real bad.